second_test/second_test.spec.js

```diff
@@ -0,0 +1,37 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+test('Perform login', async ({ page }) => {
+	await page.goto('https://www.skroutz.gr/');
+	//await page.getByRole('link', { name: 'Σύνδεση / Εγγραφή' }).click();
+	await page.click('a.icon');
+	await page.getByPlaceholder('Email').click();
+	await page.getByPlaceholder('Email').fill('polegar85@gmail.com');
+	//await page.getByRole('button', { name: 'Συνέχεια', exact: true }).click();
+	await page.click('button.btn-signin.check-email');
+	
+	await page.getByPlaceholder('Συνθηματικό').click();
+	await page.getByPlaceholder('Συνθηματικό').fill('asdfasdf1');
+	
+	//await page.getByRole('button', { name: 'Σύνδεση' }).click();
+	await page.click('button.btn-signin.password-submit');
+});
+
+
+test.only('Add two items to basket', async ({ page }) => {
+	await page.goto('https://www.skroutz.gr/');
+	const locator = page.locator('input#search-bar-input');
+	await locator.fill('keyboard');
```

Comment:
In general, for better readability, group your commands that fit together and use line breaks, for example:

```
await page.goto('https://www.skroutz.gr/');

const locator = page.locator('input#search-bar-input');
await locator.fill('keyboard');
```